releases/release-1.22/release-notes/major-themes.md

```diff
@@ -0,0 +1,59 @@
+
+### External credential providers
+
+[client-go credential plugins](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#client-go-credential-plugins) have been in beta since 1.11, a few eons ago. With the release of Kubernetes 1.22, this feature set graduates to stable. The GA feature set includes improved support for plugins that provide interactive login flows. This release also contains a number of bug fixes to the feature set. Aspiring plugin authors can look at [sample-exec-plugin](https://github.com/ankeesler/sample-exec-plugin) as a way to get started.
+Related to this topic, [the in-tree azure and gcp auth plugins have been deprecated](https://github.com/kubernetes/kubernetes/pull/102181) in favor of out-of-tree implementations of these plugins.
+
+### SIG Windows development tools
+
+To grow the developer community, sig-windows released [multiple tools](https://github.com/kubernetes-sigs/sig-windows-dev-tools/), which supports multiple CNI providers (antrea, calico), can run on multiple platforms (any vagrant compatible provider, such as hyper-v, virtualbox, vsphere), and provides a sandbox for running bleeding edge windows features from scratch by compiling the windows kubelet and kube proxy and running from sig-release daily builds.
+
+### SIG Cluster Lifecycle adds features to kubeadm
+
+A new [v1beta3 configuration API](https://github.com/kubernetes/kubeadm/issues/1796). It [iterates over v1beta2](https://pkg.go.dev/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta3) by adding some long requested features and deprecating some existing ones. The `V1beta3` is now the preferred API version, but `v1beta2` is not yet deprecated (TBD in a future release).
+
+[A new Alpha feature](https://github.com/kubernetes/kubeadm/issues/2473) for running the kubeadm control plane components as non-root users. This is a long requested security measure in kubeadm. To try it you must enable the kubeadm-specific `RootlessControlPlane` feature gate. After we improved the sleep for security folks, they can drink their morning cup of beverage a bit less hastily.
+
+### Lets `kubectl debug` with the new and improved API for Ephermal Containers
+
+The API used to create Ephemeral Containers changed in 1.22. The Ephemeral Containers feature is alpha anddisabled by default, and the new API will not work with clients attempting to use the old API.
+Users who create ephemeral containers using kubectl debug should note that kubectl version 1.22 will attempt to fallback to the old API, but older versions of kubectl will not work with cluster versions of 1.22 or later. Please update kubectl to 1.22 if you wish to use kubectl debug with a mix of cluster versions. (This only applies to debugging using ephemeral containers.)
```

Comment:
Not sure this is a major theme? More like a “you must read this before you upgrade” detail.
Anyway, here's how I'd word it.
```suggestion
For stable features, the `kubectl` tool follows the Kubernetes [version skew policy](https://kubernetes.io/releases/version-skew-policy/);
however, kubectl v1.21 and older do not support the new API for ephemeral containers.

Users who create ephemeral containers using `kubectl debug` should note that kubectl version 1.22 will attempt to fall back to the old API; older versions of kubectl will not work with cluster versions of 1.22 or later. Please update kubectl to 1.22 if you wish to use `kubectl debug` with a mix of cluster versions.
```